### PR TITLE
Header design

### PR DESCRIPTION
**Pull request by dutchmartin**
Start with recreating the design from #15
Depends on #2

### CODE REVIEW

File: src/components/atoms/ActionBotton.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import styled from "styled-components";
+
+const Button = styled.div`
```
Comment (Kingdutch) [inline src/components/atoms/ActionBotton.js]:
```suggestion
const ActionButton = styled.a`
  display: inline-block;
```

Making this an anchor tag allows the button to be exported as is without creating a separate wrapping React element.

File: src/components/organisems/Header.js [hunk 2]
```diff
@@ -26,8 +29,13 @@ const Title = styled.div`
...
 
 const HeaderLink = styled(GatsbyLink)`
-   color: #2b6cb0;
+   color: #fff;
    text-decoration: none;
+   margin: 0 5px 0 5px;
+   &:visited {
+    color: #fff;
+    text-decoration: none;
+   }
```
Comment (Kingdutch) [inline src/components/organisems/Header.js]: Can be simplified to:

```
margin: 0 5px;
&, &:visited {
  color: inherit;
  text-decoration: none;
}
```

This way changing the color in the header will also change the link color

File: src/components/organisems/Header.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 import { Link as GatsbyLink } from 'gatsby';
 import styled from "styled-components";
+import ActionBotton from "../atoms/ActionBotton";
 
 const HeaderWrapper = styled.div`
   margin-bottom: 5px;
-  background-color: rgba(200,216,217,.3);
+  background-color: #211a74;
+  color: #fff;
+  display: flex;
```
Comment (Kingdutch) [inline src/components/organisems/Header.js]: I should've added comments but the `HeaderWrapper` is for the full-width background color with the `HeaderContainer` having the centered header content. So this wrapper probably doesn't need `display: flex;`

File: src/components/organisems/Header.js [hunk 4]
```diff
@@ -45,6 +53,7 @@ export default function Header({ siteTitle, isFront }) {
           <HeaderLink className={"focus:underline hover:underline"} to="/about">About</HeaderLink>
         </NavBar>
       </HeaderContainer>
+        <ActionBotton/>
```
Comment (Kingdutch) [inline src/components/organisems/Header.js]: Moving this inside of the `NavBar` component will probably put it where you want it. 

There's also a typo in the component and filename: `ActionBotton` -> `ActionButton`.

Moving the `href` and `text` here: `<ActionButton href="csp.rustfest.eu">Submit Talk</ActionButton>` will allow us to re-use the button in other places.

**Review by Kingdutch — COMMENTED** [review]
Cool stuff! I've added some comments on where we can make improvements 😃

File: src/components/organisems/Header.js (hunk 3 above)
Comment (dutchmartin) [reply src/components/organisems/Header.js]: I am not sure what to do with it, but I use the display flex to align the button in the correct place.

**Comment by Kingdutch** [comment]
Rebased this based on the changes in `feature/team-page`

**Comment by Kingdutch** [comment]
I moved the Call to Action inside of the navbar so that it properly sits within the page and doesn't float all the way to the right on large screens. I've also changed the spacing a bit so that adding more links will work properly.

image

**Review by Kingdutch — APPROVED** [review]

**Pull request closed without merging**